vsts/pipelines/PlatformBinaries/php.yml

```diff
@@ -4,14 +4,25 @@ variables:
 stages:
 - stage: Build
   jobs:
-  - job: Php
+  - job: Php_Stretch_SDK 
```

Comment:
Seems like you might need to fix in few other places for this to work as expected.

Below link will give you the list of blobs in the 'python' container:
https://oryxsdksdev.blob.core.windows.net/python?restype=container&comp=list&include=metadata

I am highlighting the missing details in the new uploaded tarball when compared to existing tarballs

![image](https://user-images.githubusercontent.com/6559784/99465593-99f1fd80-28ef-11eb-97be-745e1dd021f7.png)

Reply:
Thanks for pointing that out .. I will rectify it .. good thing is I got a successful build after some tries :) .. this